src/domain/banner/bannerHero/BannerHero.tsx

```diff
@@ -135,6 +138,20 @@ const BannerHero: React.FC<Props> = ({ banner, location }) => {
           }}
         />
       )}
+      {heroImageCredits && (
+        <div
+          className={classNames(
+            styles.imageCredits,
+            titleAndDescriptionColor &&
```

Comment:
This is repeated twice, could be a const
`const titleAndDescriptionStyle= titleAndDescriptionColor && styles[color${capitalize(titleAndDescriptionColor)}]`